src/main/kotlin/org/jetbrains/plugins/bsp/magicmetamodel/impl/DefaultMagicMetaModelState.kt

```diff
@@ -2,28 +2,9 @@ package org.jetbrains.plugins.bsp.magicmetamodel.impl
 
 import org.jetbrains.bsp.protocol.AndroidTargetType
 import org.jetbrains.plugins.bsp.magicmetamodel.MagicMetaModelTemporaryFacadeState
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.AndroidAddendum
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.BuildTargetId
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.BuildTargetInfo
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.ContentRoot
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.GenericModuleInfo
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.GenericSourceRoot
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.IntermediateLibraryDependency
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.IntermediateModuleDependency
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.JavaAddendum
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.JavaModule
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.JavaSourceRoot
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.KotlinAddendum
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.Library
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.Module
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.ModuleCapabilities
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.PythonLibrary
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.PythonModule
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.PythonSdkInfo
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.ResourceRoot
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.ScalaAddendum
-import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.includesPython
+import org.jetbrains.plugins.bsp.magicmetamodel.impl.workspacemodel.*
```

Comment:
gwiazdka niefajna, da się to łatwo wyłączyć w ij https://www.baeldung.com/intellij-disable-wildcard-import